### PR TITLE
Create fees-and-funding-for-non-UK-trainees.md

### PR DESCRIPTION
**Pull request by laurahermionetennant92**
New page created for international fees and funding

### CODE REVIEW

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 1]
```diff
@@ -0,0 +1,321 @@
+---
+title: "Training fees and financial support for non-UK trainee teachers"
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:2]:
```suggestion
title: "Fees and financial support for non-UK trainee teachers"
```
We could just shorten this slightly to make it more readable?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+
+
+* non-UK citizens living in the UK
+* non-UK citizens living outside the UK
+* Irish citizens
+* refugees or asylum seekers
+
+## Teacher training fees
+
+As a non-UK candidate, you can expect to pay between £9,250 and about £32,000 to train to teach in England – fees vary between training providers. You’ll also have to cover your living costs.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:66]:
```suggestion
As a non-UK candidate, you can expect to pay between £9,250 and about £32,000 to train to teach in England – fees vary between training providers.
```
Because this section is talking about teacher training fees, I wonder if we need that last sentence?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+
+Depending on your immigration status, you may get a reduction in your fees to the rate paid by UK applicants (set at a maximum of £9,250). If you are eligible, this fee will be the same for all courses offered by accredited training providers. Please note – most non-UK candidates will not be eligible for this type of financial support. 
+
+If you have achieved highly in your bachelor’s degree and have at least 3 years’ relevant work experience, you can apply for a salaried teacher training course. On this type of course, you will not pay a fee, and you can earn money while you train in a school. However, these courses are limited in number and very competitive. 
+
+[Find teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses). 
+
+
+## Financial support for non-UK trainee teachers of languages and physics
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:75]:
```suggestion
## Financial support for non-UK trainee languages and physics teachers
```

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+* scholarships worth up to £29,000 for French, German, Spanish and physics
+
+
+If you are eligible, you can receive both an international relocation payment and a bursary or scholarship. You cannot receive both a bursary and a scholarship.
+
+### Get an international relocation payment (IRP) worth £10,000
+
+The IRP is designed to help pay the relocation expenses of languages or physics teacher trainees moving to the UK. 
+
+If you are eligible, you’ll receive a cash sum of £10,000 from DfE. You will not have to pay this money back and it can be spent in any way you wish. You can only receive the IRP once.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:91]:
```suggestion
If you are eligible, you’ll receive a cash sum of £10,000 from the Department for Education. You will not have to pay this money back and it can be spent in any way you wish. You can only receive the IRP once.
```

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+#### How the IRP will be paid
+
+If you have a place on a teacher training course where you pay a fee, you do not need to apply for the IRP. Your training provider will email you if your choice of course makes you eligible for the IRP. Your visa status and date of arrival will be checked by DfE. If you meet the eligibility requirements, you’ll get the IRP around the end of your first term.
+
+If you have a place on a teacher training course where you are paid a salary to work while you train, email us at IRP.ExpressInterest@education.gov.uk to express your interest in the IRP. We’ll be in touch when IRP applications open for salaried trainees in autumn 2023. If your application is successful, you’ll get the IRP around the end of your first term.
+
+You will not need to pay an agent or any other third party to complete your application. If you have had a message asking you to apply for or contact someone about the IRP, [this could be fraudulent](https://www.gov.uk/report-suspicious-emails-websites-phishing).
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:136]: I'm not sure this link text really works - it doesn't tell the user where the link will take them. And I'm not sure if this link is necessary?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+* a PhD
+
+Each bursary scheme will have their own individual terms and conditions – contact your chosen teacher training provider for more information.
+
+The languages bursary covers all languages offered by accredited teacher training providers. If a course combines 2 subjects (for example, physics plus mathematics, or modern foreign languages plus English), at least 50% of the course must be made up of the bursary subject.
+
+
+#### Scholarships for physics, French, German and Spanish
+
+To get a scholarship, you’ll need to be a high-quality student and pass a tough selection process, run by the professional body for your subject.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:170]:
```suggestion
To get a scholarship, you’ll need to be a high-quality student and pass a tough selection process.
```
Will users understand what is meant by 'professional body'? Do they need this bit of information?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+
+[Find teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses) in languages and physics. 
+
+## Eligibility for other types of financial support
+
+Depending on your immigration status, you may be able to get other kinds of financial help from the UK government. 
+
+You are more likely to get financial support if you have some type of settled status in the UK – for example, through the EU Settlement Scheme or indefinite leave to remain. Without settled status, most non-UK candidates will not be eligible for financial support.
+
+Financial support is only available for courses where you pay a fee to study teacher training.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:201]: Is this true? I thought earlier on, you said candidates will be able to apply for the IRP if they're on a salaried course in autumn?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 8]
```diff
@@ -0,0 +1,321 @@
...
+
+* you are an Irish citizen
+* you have been living in Ireland, or Ireland and the UK, Guernsey, Jersey or the Isle of Man, throughout the 3 years before the start of your course
+
+### Support for trainee teachers  from the European Economic Area (EEA) or Switzerland
+
+If you meet both the following requirements:
+
+* you have settled status under the EU Settlement Scheme
+* you have been resident in the UK throughout the 3 years before the start of your course
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:220]:
```suggestion
* you have been a resident in the UK throughout the 3 years before the start of your course
```

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 9]
```diff
@@ -0,0 +1,321 @@
...
+
+you will generally be eligible for a:
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:222]: I wonder if this whole section is clear enough? Might there be a better way to separate this info out using subheadings somehow?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 10]
```diff
@@ -0,0 +1,321 @@
...
+If you are an asylum seeker you will not be eligible for financial support, but if you are recognised as a refugee after your course has started, you may be able to claim financial support to complete your studies.
+
+Financial support can include:
+
+* help with living expenses (a maintenance loan)
+* help with course fees (a tuition fee loan)
+
+If, as a refugee, you are eligible for help with course fees, you may also be eligible for a bursary or scholarship.
+
+## Types of financial support available
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:267]: I wonder if this whole section should come further up the page? Makes more sense to me to explain what these types of support are first before explaining who is eligible for them

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 11]
```diff
@@ -0,0 +1,321 @@
...
+### Home fee rate
+If you meet the requirements for a tuition fee loan, or tuition fee loan and maintenance loan, you will be charged fees at the rate set for UK candidates (set at a maximum of £9,250 for a full-time course).
+
+### Bursaries or scholarships 
+Physics and languages bursaries and scholarships are open to all qualifying non-UK candidates, regardless of their immigration status.
+However, other bursaries and scholarships are only available to candidates who meet the eligibility requirements for a tuition fee loan, or tuition fee and maintenance loan.
+If you are awarded a bursary or scholarship, the money is paid directly into your bank account and you can use it as you wish. You do not have to pay this money back.
+
+
+[Learn more about bursaries and scholarships in other subjects, and how to apply](/funding-and-support).
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:286]:
```suggestion
[Learn more about bursaries and scholarships in other subjects, and how to apply](/funding-and-support/scholarships-and-bursaries). 
```

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 12]
```diff
@@ -0,0 +1,321 @@
...
+[Apply to Student Finance England](https://www.gov.uk/student-finance) for:
+
+* tuition fee loans
+* maintenance loans
+* extra help if you have a disability, children or an adult dependant
+
+Talk to your training provider about:
+
+* home fee rate
+* a [bursary or scholarship](/funding-and-support)
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:311]:
```suggestion
* a [bursary or scholarship](/funding-and-support/scholarships-and-bursaries)
```

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 13]
```diff
@@ -0,0 +1,321 @@
...
+
+## Get help with financial support
+
+[Student Finance England](https://www.gov.uk/contact-student-finance-england) has a dedicated service for EU students and for EEA citizens working in the UK.
+
+Citizens from other countries can find more detailed information at [Student finance for undergraduates: eligibility](https://www.gov.uk/student-finance/who-qualifies).
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:317]:
```suggestion
Citizens from other countries can also [find out more about eligibility for student finance](https://www.gov.uk/student-finance/who-qualifies).
```

File: app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md [hunk 14]
```diff
@@ -73,9 +73,51 @@ If you are already a qualified teacher, you do not need to retrain to teach in E
...
 * a visa or immigration status allowing you to work in the UK
 
-### Your professional qualifications 
+## Get an international relocation payment (IRP) worth £10,000
+
+If you are interested in moving to the UK to teach languages or physics in an English secondary school, you could be eligible for a cash sum of £10,000 from the UK government to help with moving expenses, including visa costs and the [immigration health surcharge](https://www.gov.uk/healthcare-immigration-application). 
+
+The international relocation payment (IRP) will be available to non-UK citizens starting work in English schools in the academic year September 2023 to July 2024. You will not have to pay this money back and it can be spent in any way you wish. You can only receive the IRP once.
+
+### Check your eligibility for the IRP
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md:82]: If all the following info is on the new fees and funding page, shouldn't we just be linking there rather than repeating it here? Couldn't we add a link along the lines of 'Find out more about the IRP' and include a jumplink to that section of the page?

File: app/views/content/non-uk-teachers/ukraine.md [hunk 15]
```diff
@@ -117,9 +126,16 @@ You can also learn English with a private provider. Well respected English langu
...
 
-[Find out more about teaching in England if you trained overseas](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
+### Apply for QTS in England
+
+From 1 February 2023, qualified teachers from Ukraine who [meet the criteria for QTS](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers) will be able to [apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start) without paying a fee or undergoing more training. 
+
+### Assessment only QTS
+If you find you don't meet the eligibility criteria to apply for QTS, you could consider assessment only QTS. This means you can work in an English school and, alongside your job, apply to an [English teacher training provider](/assessment-only-providers) who will assess if you meet the standards for English QTS. DfE has made funding available for candidates who apply for assessment only courses. Check assessment only training providers for more information about eligibility and funding.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/ukraine.md:138]:
```suggestion
If you find you do not meet the eligibility criteria to apply for QTS, you could consider assessment only QTS. This means you can work in an English school and, alongside your job, apply to an English teacher training provider who will assess if you meet the standards for English QTS.

DfE has made funding available for candidates who apply for assessment only courses. Check with [assessment only training providers]((/assessment-only-providers) for more information about eligibility and funding.
```
Thought it made sense to break up this big block of text, and also move the link. It didn't feel like the right link text to take them to that page for me? Hope this makes sense

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 16]
```diff
@@ -0,0 +1,321 @@
...
+### Maintenance loan
+Money is paid directly into your bank account to help with your living costs. This is a loan you have to pay back, but you’ll only start your repayments when your income is over a certain amount – for the 2023 to 2024 financial year, this is £24,996 per year.
+
+### Home fee rate
+If you meet the requirements for a tuition fee loan, or tuition fee loan and maintenance loan, you will be charged fees at the rate set for UK candidates (set at a maximum of £9,250 for a full-time course).
+
+### Bursaries or scholarships 
+Physics and languages bursaries and scholarships are open to all qualifying non-UK candidates, regardless of their immigration status.
+However, other bursaries and scholarships are only available to candidates who meet the eligibility requirements for a tuition fee loan, or tuition fee and maintenance loan.
+If you are awarded a bursary or scholarship, the money is paid directly into your bank account and you can use it as you wish. You do not have to pay this money back.
```
Comment (EllieNodder) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:283]: Do we need to put this info onto a new line just to break up this big block of text?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md (hunk 5 above)
Comment (gemmadallmandfe) [reply app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:136]: Agree - this link text does not meet accessibility standards so needs to be changed (or removed entirely)

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md [hunk 17]
```diff
@@ -0,0 +1,321 @@
...
+
+* you are an Irish citizen
+* you have been living in the UK, Guernsey, Jersey or the Isle of Man throughout the 3 years immediately before the start of your course
+
+You will generally be eligible for a tuition fee loan if you meet both the following requirements:
+
+* you are an Irish citizen
+* you have been living in Ireland, or Ireland and the UK, Guernsey, Jersey or the Isle of Man, throughout the 3 years before the start of your course
+
+### Support for trainee teachers  from the European Economic Area (EEA) or Switzerland
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:215]:
```suggestion
### Support for trainee teachers from the European Economic Area (EEA) or Switzerland
```

File: app/views/content/non-uk-teachers/ukraine.md [hunk 18]
```diff
@@ -117,9 +126,16 @@ You can also learn English with a private provider. Well respected English langu
...
 * [C2 Proficiency, formerly known as Cambridge English: Proficiency (CPE)](https://www.cambridgeenglish.org/exams-and-tests/proficiency/)
 
+### Qualified teacher status
 
+[Qualified teacher status (QTS)]((https://www.gov.uk/guidance/qualified-teacher-status-qts)) is the professional status teachers in England gain at the end of their teacher training. Schools use QTS as a measure of teaching skills and experience, so you will find it much easier to get a teaching job in England if you have QTS.
 
-[Find out more about teaching in England if you trained overseas](/non-uk-teachers/teach-in-england-if-you-trained-overseas). 
+### Apply for QTS in England
+
+From 1 February 2023, qualified teachers from Ukraine who [meet the criteria for QTS](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers) will be able to [apply for QTS in England](https://apply-for-qts-in-england.education.gov.uk/eligibility/start) without paying a fee or undergoing more training.
```
Comment (gemmadallmandfe) [inline app/views/content/non-uk-teachers/ukraine.md:135]: This sentence needs rewriting now that 1 Feb is in the past?

File: app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md (hunk 7 above)
Comment (philspray) [reply app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md:201]: Reword this to say tuition fees and loans
